layouts/partials/head.html

```diff
@@ -29,10 +29,14 @@
   {{ $customCssPath := "css/custom.css" }}
   {{ $customCss:= resources.Get $customCssPath | resources.ExecuteAsTemplate $customCssPath . }}
   <link rel="stylesheet" type="text/css" href="{{ $customCss.RelPermalink }}">
+  
+  {{ if and (hugo.IsProduction) }}
```

Comment:
No need for and here

```suggestion
  {{ if hugo.IsProduction }}
```